aws-datastore/src/main/java/com/amplifyframework/datastore/storage/LocalStorageAdapter.java

```diff
@@ -156,6 +156,20 @@ <T extends Model> void delete(
             @NonNull Consumer<DataStoreException> onError
     );
 
+    /**
+     * Batches operations for a given type in a single transaction.
+     *
+     * @param <T>        The type of items being saved/deleted
+     * @param operations A list of operations that will sequentially execute
+     * @param onComplete A callback that will be invoked when the transaction batch succeeds
+     * @param onError    A callback that will be invoked when any operations fails with an error
+     *                   If an error is received, the entire transaction will be unsuccessful
+     */
+    <T extends Model> void batchSyncOperations(
```

Comment:
Still need to make sure this is allowed. This is on a public interface that should not be public (and would never be expected for the customer to extend from.

Reply:
It's a public interface so I don't think it is.